docs/install_guides/installing-openvino-from-archive-windows.md

```diff
@@ -39,74 +38,72 @@ You can also check the [Release Notes](https://software.intel.com/en-us/articles
+tar -xf openvino_2022.2.0.7713.zip
+ren w_openvino_toolkit_windows_2022.2.0.7713.af16ea1d79a_x86_64 openvino_2022.2.0.7713
+```
 
-The core components are now installed. Continue to the next section to configure the environment.
+Finally, create a symbolic link to the folder by issuing:
+```sh
+mklink /D openvino_2022 openvino_2022.2.0.7713
+```
 
-### <a name="set-the-environment-variables"></a>Step 2: Configure the Environment
+> **NOTE**: If you have already installed a previous release of OpenVINO 2022, a symbolic link to the `openvino_2022` folder may already exist. If you want to override it, nagivate to the `C:\Program Files (x86)\Intel` folder and delete the existing linked folder before running the `mklink` command.
+
+Congratulations, you finished installation! The C:\Program Files (x86)\Intel\openvino_2022 folder now contains the core components for OpenVINO™. When other pages in OpenVINO™ documentation refer to the `<INSTALL_DIR>` directory, this is the folder they're referring to. If you installed OpenVINO™ in a different location, make sure to use that instead.
 
-> **NOTE**: If you installed the Intel® Distribution of OpenVINO™ to a non-default install directory, replace `C:\Program Files (x86)\Intel` with that directory in this guide's instructions.
+### <a name="set-the-environment-variables"></a>Step 2: Configure the Environment
 
-You must update several environment variables before you can compile and run OpenVINO™ applications. Open the Command Prompt, and run the `setupvars.bat` batch file to temporarily set your environment variables:
+You must update several environment variables before you can compile and run OpenVINO™ applications. Open the Command Prompt, and run the `setupvars.bat` batch file to temporarily set your environment variables. Again, if you installed OpenVINO™ in a folder other than `C:\Program Files (x86)\Intel\openvino_2022`, use that location instead.
 
 ```sh
-"<INSTALL_DIR>\setupvars.bat"
+"C:\Program Files (x86)\Intel\openvino_2022\setupvars.bat"
 ```
 
-**Optional**: OpenVINO™ toolkit environment variables are removed when you close the command prompt window. You can permanently set the environment variables manually.
+> **Important**: The above command must be re-run every time a new Command Prompt window is opened.
 
-> **NOTE**: If you see an error indicating Python is not installed when you know you installed it, your computer might not be able to find the program. Check your system environment variables, and add Python if necessary.
+> **NOTE**: If you see an error indicating Python is not installed, Python may not be added to the PATH environment variable (as described [here](https://docs.python.org/3/using/windows.html#finding-the-python-executable)). Check your system environment variables, and add Python if necessary.
```

Comment:
To me this reads as if Python is required for using OpenVINO, but it is not, it is only required for using the Python API.

Reply:
Per our email conversation, I created Issue #13411 to report this problem.